tests/run-regressionTest.sh

```diff
@@ -20,35 +20,14 @@ ${BINPATH}/${EXE_NAME} ${TEST_ARGS} --output-dir=${RESULT_PATH}
 test $? -eq 0 || exit 1
 cd ..
 
-ecode=0
-echo "=== Executing comparison for summary file ==="
-${COMPARE_ECL_COMMAND} -t SMRY ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${ABS_TOL} ${REL_TOL}
-if [ $? -ne 0 ]
-then
-  ecode=1
-  ${COMPARE_ECL_COMMAND} -a -t SMRY ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${ABS_TOL} ${REL_TOL}
-fi
-
-ignore_extra_kw=""
-if grep -q "ignore_extra" <<< $ghprbCommentBody
-then
-    ignore_extra_kw="-x"
-fi
 
-echo "=== Executing comparison for restart file ==="
-${COMPARE_ECL_COMMAND}  ${ignore_extra_kw} ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${ABS_TOL} ${REL_TOL}
-if [ $? -ne 0 ]
-then
-  ecode=1
-  ${COMPARE_ECL_COMMAND} ${ignore_extra_kw} -a ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${ABS_TOL} ${REL_TOL}
-fi
-
-echo "=== Executing comparison for init file ==="
-${COMPARE_ECL_COMMAND} ${ignore_extra_kw} -t INIT ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${ABS_TOL} ${REL_TOL}
+ecode=0
+echo "=== Executing comparison for EGRID, INIT, UNRST and RFT files if these exists in refrece folder ==="
+${COMPARE_ECL_COMMAND}  ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${RESULT_PATH}/${FILENAME} ${ABS_TOL} ${REL_TOL}
 if [ $? -ne 0 ]
 then
   ecode=1
-  ${COMPARE_ECL_COMMAND} ${ignore_extra_kw} -a -t INIT ${RESULT_PATH}/${FILENAME} ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${ABS_TOL} ${REL_TOL}
+  ${COMPARE_ECL_COMMAND} -a  ${INPUT_DATA_PATH}/opm-simulation-reference/${EXE_NAME}/${FILENAME} ${RESULT_PATH}/${FILENAME} ${ABS_TOL} ${REL_TOL}
```

Comment:
This appears to drop support for the `-x` option that powers the CI system's "ignore_extra" option. Is that intentional?

Reply:
@bska it there anything more you need from me before this can be merged ? Please let me know.

Reply:
> Is there anything more you need from me before this can be merged?

I would really appreciate it we didn't lose support for `ignore_extra` as part of merging this. If it is too much work to restore the feature now I'll make a follow-up PR that reimplements the support.

Reply:
I have not updated the PR. The extra keyword option for compareECL is tested in EclRegressionTest.cpp, but I'm not sure how to test this in this context. Will you do this ? or is there anything more you would like me to do ?

Reply:
> I have now updated the PR.

Thanks. Much appreciated!

> The extra keyword option for `compareECL` is tested in `EclRegressionTest.cpp`, but I'm not sure how to test this in this context. Will you do this?

Yes.  I will add any requisite scaffolding to ensure that the `ignore_extra` Jenkins keyword implies what we think it does.  It's not very commonly used, but it is extremely useful from time to time.